example/main.go

```diff
@@ -55,23 +57,28 @@ func main() {
 	reg := prometheus.NewRegistry()
 	metrics := aibridge.NewMetrics(reg)
 
+	// Setup tracing
+	tracer := otel.GetTracerProvider().Tracer("exampleTracer")
```

Comment:
Oh nice, thanks for updating the example.
We should probably have a CI job to validate this example compiles.